SEC/Securities/Pools.rdf

```diff
@@ -200,5 +230,13 @@
 		<skos:definition>security that is included in a securities pool</skos:definition>
 		<fibo-fnd-utl-av:explanatoryNote>This is a pool of any kind of security, and not therefore necessarily a pool of debt, though usually it is debt securities which are pooled.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-sec-sec-pls;SpecialPurposeVehicle">
```

Comment:
I'm wondering if this is still not general enough. Maybe it should be in BE. Can't SPVs be created for other reasons than related to Pools? For example from Investopedia
What Is the Function of SPVs in Public-Private Partnerships?
Public-private partnerships are collaborations between a government agency and a privately owned company. Many private partners in public-private partnerships demand a special purpose vehicle as part of the arrangement. This is especially true for capital-intensive endeavors, such as an infrastructure project. The private company might not want to take on too much financial exposure, so an SPV is created to absorb some of the risks

Reply:
@rivettp - ok, I'll move it again.  That crossed my mind when I put it in pools, which is significantly higher up the food chain than where it started.